dts/bindings/modem/quectel,eg25-g.yaml

```diff
@@ -0,0 +1,15 @@
+description: Quectel EG25-G modem
+
+compatible: "quectel,eg25-g"
+
+include: uart-device.yaml
+
+properties:
+  mdm-reset-gpios:
+    type: phandle-array
+
+  mdm-dtr-gpios:
+    type: phandle-array
+
+  mdm-wdisable-gpios:
+    type: phandle-array
```

Comment:
Do you want any of these to be required?

Reply:
yes good idea, although the current cellular driver ignores dtr and wdisable

Reply:
Maybe you want those two optional then.